dts/bindings/i2c/ite,common-i2c.yaml

```diff
@@ -30,6 +30,25 @@ properties:
       4 = I2C_CHANNEL_E,
       5 = I2C_CHANNEL_F,
 
+  channel-switch-sel:
+    type: int
+    required: true
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+      - 4
+      - 5
+    description: |
```

Comment:
Can you add an example in the documentation on how to use the channel-switch-sel?

Specifically, the `port-num` property is fixed and should not be changed by the application.  Also, if the application modifies the `channel-switch-sel` the application must also modify the pinctrl setting.